src/data/energy.ts

```diff
@@ -271,6 +278,9 @@ const getEnergyData = async (
 
   const consumptionStatIDs: string[] = [];
   const statIDs: string[] = [];
+  const statisticIdsWithMeta: StatisticsMetaData[] = await getStatisticIds(
```

Comment:
it's expensive to always fetch all statistic IDs even it might not be necessary for most energy configurations.

Reply:
It's the same approach as in `statistics-chart.ts`, so I just followed it.
Anyway, there are some possibilites to change it, which do you think would be the best (or maybe other option)?
- save unit of measurement in `EnergyManager`
- add COLUMN `source type` to `statistics_meta` which could have value either `entity` or `external`, then filter based on source_type in webapi
- add filter include do webapi and do something like this
```javascript
const statisticIdsWithMeta: StatisticsMetaData[] = await getStatisticIds(
    hass
    include: prefs.energy_sources.filter(source => !(source.stat_energy_from in hass.states))
  );
```

Reply:
@balloob any thoughts about it?

Reply:
new WS API added https://github.com/home-assistant/core/pull/68331